blocks/monitoring/aggregation.py

```diff
@@ -138,11 +138,62 @@ def get_aggregator(self):
         return aggregator
 
 
-def mean(numerator, denominator=1.):
+def mean(numerator, denominator=1., name=None):
     """Mean of quantity (numerator) over a number (denominator) values."""
     variable = numerator / denominator
     variable.tag.aggregation_scheme = Mean(numerator, denominator)
-    variable.name = numerator.name
+    if name is not None:
```

Comment:
Not sure if I like it. One can set name to the variable returned by `mean`, which is almost as simple. Otherwise you have to add these 4 lines to every `mean`-like function, e.g. `sum_` below.